Casks/l/last-window-quits.rb

```diff
@@ -0,0 +1,24 @@
+cask "last-window-quits" do
+  version "1.1.3"
+  sha256 "2b6645a0910f27a7021828a44a8744e03db7a7a99b75c3d88ad5fc09c31f29e0"
+
+  url "https://lawand.io/wp-content/uploads/2024/11/last-window-quits-#{version}.zip"
+  name "last-window-quits"
+  desc "Automatically quit apps when their last window is closed"
+  homepage "https://lawand.io/last-window-quits/"
+
+  livecheck do
+    url "https://raw.githubusercontent.com/lawand-dot-io/last-window-quits/main/appcast.xml"
+    strategy :sparkle, &:short_version
+  end
+
+  depends_on macos: ">= :high_sierra"
```

Comment:
```suggestion
  auto_update true
  depends_on macos: ">= :high_sierra"
```

I assume based on the presence of Sparkle, that this has an in-app auto updating mechanism?

Reply:
Hi. Yes, in-app auto update using sparkle is available